### DISCUSSION TITLE
New beginner's guide ?

### DISCUSSION THREAD

**Discussion by Mas99** [iptv-org/community#335] [Q&A] [answered]
Hi All,

I am new to IPTV world. I have VLC player and i have downloaded repository from Github. I can play the file if I copy it directly to VLC network stream. 

However I don't know how to use this repository ? I can't see any grouping (by category or country ). I have to copy the M3u file each time to get the channels list.

Is there installation guide or beginner's guide I can use ? Do I need to copy the file each time I want to watch channel ?

**Comment by ghost** [comment]
Hi.

Checkout the website https://iptv-org.github.io, search in the search bar for a channel. Check for it there is any stream link and play it in vlc.

You can also check channels and you can create your IPTV playlist.

(Sorry for my bad english by the way)

**Comment by BellezaEmporium** [comment, marked as answer]
> Hi All,
> 
> I am new to IPTV world. I have VLC player and i have downloaded repository from Github. I can play the file if I copy it directly to VLC network stream.
> 
> However I don't know how to use this repository ? I can't see any grouping (by category or country ). I have to copy the M3u file each time to get the channels list.
> 
> Is there installation guide or beginner's guide I can use ? Do I need to copy the file each time I want to watch channel ?

All of the data you're looking (or wanting) to look for is said in the README.md file.

There you'll find a bit of explicative information on how we split the channels, and everything linked to the way we proceed.
